src/jayashri/A21_PrintDigits.java

```diff
@@ -0,0 +1,20 @@
+package jayashri;
+
+public class A21_PrintDigits {
+	String str = "t1e3chnoc45red9its";
+	String temp = " ";
+
+	void print() {
```

Comment:
the method should be parameterized, so that it will be dynamic for other inputs as well.